packages/kolibri-tools/package.json

```diff
@@ -99,6 +99,9 @@
     "npm": ">= 8"
   },
   "devDependencies": {
-    "rewire": "^6.0.0"
+    "rewire": "^6.0.0",
```

Comment:
Hrm, I think `rewire` is misleading here - all of the dependencies in this devDependencies section should actually be the in the `dependencies` section - kolibri-tools is a library for building, testing etc. so all its testing dependencies are actually dependencies and not just for development.

Reply:
Got it. I have made the required changes and even migrated the `rewire` package to the dependencies section.